ui/src/core/xibo-cms.js

```diff
@@ -109,6 +109,14 @@ window.XiboInitialise = function(scope, options) {
     scope = ' ';
   }
 
+  // Search for any grids on the page and render them
+  $(scope + " .XiboGrid").each(function() {
```

Comment:
Can you replace the double quotes with single quotes here please (https://eslint.org/docs/latest/rules/quotes)?